src/elements/content-explorer/ItemGridThumbnail.scss

```diff
@@ -0,0 +1,27 @@
+.bce-ItemGridThumbnail {
+    margin-bottom: 10px;
+    // padding is used to maintain cell's aspect ratio on resize of container
+    padding-bottom: 65%;
+    position: relative;
+}
+
+.bce-ItemGridThumbnail-item {
+    bottom: 0;
+    left: 0;
+    margin: auto;
+    position: absolute;
+    right: 0;
+    top: 0;
+}
+
+.bce-ItemGridThumbnail-itemIcon {
```

Comment:
Do these styles need to be separate or can they all simply live on `-item` together?